spanner/spanner_samples.rb

```diff
@@ -968,6 +968,38 @@ def delete_using_partitioned_dml project_id:, instance_id:, database_id:
   # [END spanner_dml_partitioned_delete]
 end
 
+def update_using_batch_dml project_id:, instance_id:, database_id:
+  # [START spanner_dml_batch_update]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+  insert_statement = "INSERT INTO Albums "\
+    "(SingerId, AlbumId, AlbumTitle, MarketingBudget) "\
+    "VALUES (1, 3, 'Test Album Title', 10000)"
+  update_statement = "UPDATE Albums "\
+    "SET MarketingBudget = MarketingBudget * 2 "\
+    "WHERE SingerId = 1 and AlbumId = 3"
+  row_counts = Array.new
```

Comment:
Change to `row_counts = nil`
Keep it grouped together with `client.transaction do |transaction|`.

Reply:
Done.